src/grid.hoc.js

```diff
@@ -1,26 +1,21 @@
 // @flow
 import React from 'react'
-import {
-  compact,
-  getDisplayName,
-  getJustify,
-  getAlignment,
-  type Offset,
-  type Size,
-} from './utils'
+import { compact, getDisplayName, getJustify, getAlignment } from './utils'
+import { type Offset, type Size } from './types'
 import { ALIGN, JUSTIFY } from './values'
 
 export default function Grid(Component: any) {
   return class withGrid extends React.PureComponent {
     props: {
       align?: Symbol,
-      bottom?: boolean,
+      bottom: boolean,
```

Comment:
Oh I thought because default props set it it would never be not set. is that a correct assumption or does this just rely on statically checking if the values are passed?